.travis.yml

```diff
@@ -4,3 +4,19 @@ node_js:
   - '0.10'
   - '0.12'
   - 'iojs'
+
+cache:
+  directories:
+    - node_modules
+
+matrix:
+  fast_finish: true
```

Comment:
`fast_finish` makes Travis already report that it's finished, even if the builds marked as allowed failures aren't done yet. We don't have any allowed failures at the moment so this doesn't make much sense. And if we would have any allowed failures, this improvement would be tiny, and I honestly like still running everything before reporting.